views/login.njk

```diff
@@ -1,325 +1,316 @@
-                                        class="absolute right-3 top-1/2 transform -translate-y-1/2 text-base-content/40 hover:text-base-content transition-colors"
-                                        onclick="togglePassword('login-password')"
-                                    >
-                                        <i id="login-password-show" data-lucide="eye" class="h-5 w-5"></i>
-                                        <i id="login-password-hide" data-lucide="eye-off" class="h-5 w-5 hidden"></i>
-                                    </button>
-                                </div>
-                            </div>
-
-                            <!-- Remember Me & Forgot Password -->
-                            <div class="flex items-center justify-between">
-                                <div class="form-control">
-                                    <label class="label cursor-pointer gap-2">
-                                        <input type="checkbox" id="remember-me" name="remember" class="checkbox checkbox-primary checkbox-sm"/>
-                                        <span class="label-text text-sm">{{ t('auth.login.rememberMe') }}</span>
-                                    </label>
-                                </div>
-                                <a href="#" class="link link-primary text-sm">{{ t('auth.login.forgotPassword') }}</a>
-                            </div>
-
-                            <!-- Login Button -->
-                            <div class="form-control mt-6">
-                                <button type="submit" class="btn btn-primary gap-2">
-                                    <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-                                        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M11 16l-4-4m0 0l4-4m-4 4h14m-5 4v1a3 3 0 01-3 3H6a3 3 0 01-3-3V7a3 3 0 013-3h7a3 3 0 013 3v1"/>
-                                    </svg>
-                                    {{ t('auth.login.button') }}
-                                </button>
-                            </div>
-                        </form>
-                    </div>
+                <a href="#" class="link link-primary text-sm">{{ t('auth.login.forgotPassword') }}</a>
+              </div>
+              <!-- Login Button -->
+              <div class="form-control mt-6">
+                <button type="submit" class="btn btn-primary gap-2">
+                  <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                    <path
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      stroke-width="2"
+                      d="M11 16l-4-4m0 0l4-4m-4 4h14m-5 4v1a3 3 0 01-3 3H6a3 3 0 01-3-3V7a3 3 0 013-3h7a3 3 0 013 3v1"/>
+                  </svg>
```

Comment:
Hmmm.... were the instruction files not picked up?
Tagging @AnnaMcCormick who may help